packages/status/src/class-status.php

```diff
@@ -73,6 +73,34 @@ public function is_offline_mode() {
 		return $offline_mode;
 	}
 
+	/**
+	 * Is Jetpack in "No User test mode"?
+	 *
+	 * This will make jetpack act as if there were no connected users, but only a site connectino (aka blog token)
+	 *
+	 * @since 9.2.0
+	 *
+	 * @return bool Whether Jetpack's No User Testing Mode is active.
+	 */
+	public function is_no_user_testing_mode() {
+		$test_mode = false;
+		if ( defined( 'JETPACK_NO_USER_TEST_MODE' ) ) {
+			$test_mode = JETPACK_NO_USER_TEST_MODE;
+		}
+
+		/**
+		 * Filters Jetpack's No User test mode.
+		 *
+		 * @since 9.2.0
+		 *
+		 * @param bool $offline_mode Is Jetpack's offline mode active.
+		 */
+		$test_mode = (bool) apply_filters( 'jetpack_offline_mode', $test_mode );
```

Comment:
I don't think we need to document that filter here, since it's already documented:
https://github.com/Automattic/jetpack/blob/e401719b7989df671cb56d16f2320d478e81ad69/packages%2Fstatus%2Fsrc%2Fclass-status.php#L61-L71

```suggestion
		/** This filter is already documented in packages/status/src/class-status.php */
		$test_mode = (bool) apply_filters( 'jetpack_offline_mode', $test_mode );
```

Reply:
I'm sorry, actually I have to change the name of the filter! Thanks for catching this!

Reply:
fixed